app/lib/theme/variable_font_weight.dart

```diff
@@ -0,0 +1,89 @@
+// SPDX-FileCopyrightText: 2025 Phoenix R&D GmbH <hello@phnx.im>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import 'package:flutter/widgets.dart';
+
+/// Font weight for variable fonts.
+///
+/// Misnomer: Technically, this is a font style.
+class VariableFontWeight {
+  /// Thin, the least thick.
+  static const TextStyle w100 = TextStyle(
```

Comment:
I'm wondering if we shouldn't give them names instead of numbers. Assuming we want to tweak the actual weight going forward, it might be confusing to have a style that is called `w100` when the wight is really say 120.
But I don't have a strong opinion on this and I'll approve anyway.

Reply:
Below, there are also names for `normal`, `medium`, `semiBold`, and `bold`. I took the weights and names from the standard class `FontWeight` and added `medium` and `semiBold`.